easybuild/easyconfigs/i/I-TASSER/I-TASSER-4.0.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = "PackedBinary"
+
+name = "I-TASSER"
+version = '4.0'
+
+homepage = 'http://zhanglab.ccmb.med.umich.edu/I-TASSER/'
+description = """I-TASSER is a set of pre-compiled binaries and scripts for protein structure and function
+modelling and comparison."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Can't download from the web site as the file is behind a rego wall.
```

Comment:
@boegel: Sorry, "rego" is Australian and New Zealand shorthand for "registration". I've now clarified the comment.